inc/datacollector/class-user.php

```diff
@@ -95,6 +97,23 @@ public function updateAllUsersMetadata(): \Generator {
 		}
 	}
 
+	/**
+	 * Updates the network manager's site meta data
+	 *
+	 * @return void
+	 */
+	public function updateNetworkManagers(): void {
```

Comment:
wondering if it should be in pressbooks network analytics plugin, since the network admins management is mostly there, additionally this method seems to be used only there. What do you think?

Reply:
Could be an option, yes. I opted to add it here to keep it close to the other data collectors.

Reply:
We are adding a new namespace with @arzola `\PressbooksNetworkAnalytics\DataCollector` for the new table and collection actions. Probably a new method or class in that namespace would be more centralized. We can keep it here in the meantime, move to a pressbooks network analytics plugin (probably a temporary namespace method) or wait until the new namespace is created in near future PRs. I would choose option 2, but it is up to you, for me any of those options are ok. What do you think guys? @arzola  @fdalcin ?

Reply:
I'm ok with moving to network analytics once we have the new namespace. Keep in mind that user and book data collectors will continue to live here though.